src/setup/cortx_deploy

```diff
@@ -110,6 +111,46 @@ node_id=`cat /etc/machine-id`
 cluster_id=$(conf_get $(yaml_config $CLUSTER_INFO) "cluster>id")
 log $INFO "Starting provisioning on node $node_id which is part of cluster $cluster_id."
 
+deploy=false
+upgrade=false
+# Check current image version
+CONFSTOR_FILE=$(awk -F:// '{print $2}' <<< $CORTX_CONFSTORE)
+# check if confstore file is empty or not present
+if [ ! -s "$CONFSTOR_FILE" ]; then
```

Comment:
don't mention confstore as file, keep it as CONFSTORE. remove 'file' references for confstore from this script. also, can we validate the location of confstore without splitting the URL?

Reply:
Rmoved the check for consfstore as a file all-together.